components/TARGET_PSA/TARGET_TFM/mbed_lib.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "tfm",
+    "macros": [
+        "TFM_PSA_API", "MBED_TZ_DEFAULT_ACCESS=1"
+    ],
+    "config": {
+        "level": {
+            "help": "TFM security level",
```

Comment:
What is this suppose to mean?

Reply:
This is one TFM properties
security level is corresponding with PSA memory separation level desired

This is an internal TFM config, and a prep for the future when we will have a greater level of memory separation.